R/derive_params_growth_age.R

```diff
@@ -283,6 +282,10 @@ derive_params_growth_age <- function(dataset,
     ungroup()
 
 
+  by_exprs <- enexpr(by_vars)
+  by_antijoin <- setNames(as.character(by_exprs), as.character(by_exprs))
+  unmatched_records <- anti_join(dataset, added_records, by = by_antijoin)
```

Comment:
@rossfarrugia @Minlei0201 Needed an `anti_join()` object here to retain/grab records that don't have matches to the metadata, to then set appropriate PARAMCD records to indicate they had no match which then gets row binded back in, the tests reflect that as well now